hosts/azure/jenkins-controller/configuration.nix

```diff
@@ -136,6 +136,9 @@ in
       "-Dcasc.jenkins.config=${jenkins-casc}"
       # Increase the number of rows shown in Stage View (default is 10)
       "-Dcom.cloudbees.workflow.rest.external.JobExt.maxRunsPerJob=32"
+      # Allow checkouts done from local repositories in Jenkins
+      # Allows project development without need of constant deployments
+      "-Dhudson.plugins.git.GitSCM.ALLOW_LOCAL_CHECKOUT=true"
```

Comment:
I'd feel more comfortable if this was not enabled for *all environments*, but triggered through some environment variable, file made available through cloud-init, ...

Reply:
Overall, we could have common configuration for all environments, and some variations for separate environments. Or even git branches for separate environment.

I wanted to store this somewhere as I am going to delete my personal development VM soon.
If you feel too uncomfortable, we do not need to merge this.
Change does not really belong to current production environment.

Reply:
Then let's put this in some of our markdown files, and not merge this.